src/Compilers/Core/Portable/Collections/ImmutableArrayExtensions.cs

```diff
@@ -63,6 +64,23 @@ public static ImmutableArray<T> AsImmutableOrNull<T>(this IEnumerable<T> items)
             return ImmutableArray.CreateRange<T>(items);
         }
 
+        public static string PublicKeyToString(this ImmutableArray<byte> key)
```

Comment:
Probably don't want this as a general purpose extension method on `ImmutableArray<byte>`. Feels pretty specific to a scenario.